Source/WebKit/NetworkProcess/NetworkProcess.cpp

```diff
@@ -332,6 +332,9 @@ void NetworkProcess::initializeNetworkProcess(NetworkProcessCreationParameters&&
 
     setPrivateClickMeasurementEnabled(parameters.enablePrivateClickMeasurement);
     m_ftpEnabled = parameters.ftpEnabled;
+#if ENABLE(BUILT_IN_NOTIFICATIONS)
+    DeprecatedGlobalSettings::setBuiltInNotificationsEnabled(parameters.builtInNotificationsEnabled);
```

Comment:
Let's just add a bool to NetworkProcess rather than adding use of DeprecatedGlobalSettings in the network process.